packages/design-picker/src/components/index.tsx

```diff
@@ -119,6 +124,12 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 						<span className="design-picker__option-name">{ designTitle }</span>
 					) }
 					{ design.is_premium && premiumBadge }
+					{ ! premiumBadge && (
```

Comment:
This code feels strange to me. In order to start using this we would need to pass `null` as a value for `premiumBadge`? 
Maybe this would be better gated by the feature flag we are using for the new design? 🤔

Reply:
I understand your concerns about later coming around to remove the old code. Having said that, a good pattern we follow when developing big features is for them to be able to be easily turned on/off. What do you think of enabling this component on the current Design Picker? maybe just flagging the svg is enough. 

If this feels strange for other reasons, do what you think is best 👍.

Reply:
Yep, let's try and keep everything that's part of this project toggleable by the same flag.